atc/db/migration/migrations/1595261696_add_parent_build_and_job_id_index.up.sql

```diff
@@ -0,0 +1,4 @@
+BEGIN;
+  CREATE INDEX parent_job_id ON pipelines USING btree (parent_job_id);
+  CREATE INDEX parent_build_id ON pipelines USING btree (parent_build_id);
```

Comment:
@aoldershaw brought up a really good point that the `parent_build_id` index will probably never need to be used.  https://github.com/concourse/concourse/pull/5854#discussion_r461695769 We can remove it!